src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.Linux.cs

```diff
@@ -19,102 +19,167 @@ public unsafe partial class RaspberryPi3Driver : GpioDriver
         private UnixDriver _sysFSDriver = null;
         private readonly IDictionary<int, PinMode> _sysFSModes = new Dictionary<int, PinMode>();
 
-        protected override void Dispose(bool disposing)
+        private void Initialize()
```

Comment:
I know that re-ordering makes sense in terms of where each method should be declared, but it makes changes like this really hard to review as the diffs are all messed up. If you had actual changes to the implementation, do you mind just pushing those for now and keeping the same method ordering for now? We can later do a PR for reordering where all we do is that, so we don't have to review the whole file.

Reply:
Just updated.  Not sure if the this fixes your feedback.  Let me know and I'll try again.  Thx